### PR TITLE
Major changes. Multiple and complex sorting

### PR DESCRIPTION
**Pull request by patrickhacens**
Enabling complex property sorting and multiple sorts

This resolves issues #3 and #4

### CODE REVIEW

File: src/Core/PaginationData.cs [hunk 1]
```diff
@@ -1,70 +1,65 @@
...
+    public int PageSize { get; set; }
 
-        /// <summary>
-        /// Total number of pages
-        /// </summary>
-        public int PageCount => PageSize == 0 ? 0 : (int)Math.Ceiling(Total / (double)PageSize);
+    /// <summary>
+    /// Sequence of sorting definitions in which data has been sorted into
+    /// </summary>
+    public SortingDefinition[] Sorting { get; set; }
```
Comment (Ander02) [inline src/Core/PaginationData.cs:24]: Should be IEnumerable?

File: src/Core/Paginator.Core.csproj [hunk 2]
```diff
@@ -1,28 +1,37 @@
...
-  </PropertyGroup>
+	<PropertyGroup>
+		<TargetFramework>netstandard2.1</TargetFramework>
+		<AssemblyName>Nudes.Paginator.Core</AssemblyName>
+		<RootNamespace>Nudes.Paginator.Core</RootNamespace>
+		<LangVersion>11</LangVersion>
+		<Authors>Patrick Ens</Authors>
+		<Company>Nudes</Company>
+		<PackageLicenseFile>LICENSE</PackageLicenseFile>
+		<PackageProjectUrl>https://github.com/nudes-senaisp/Paginator</PackageProjectUrl>
```
Comment (Ander02) [inline src/Core/Paginator.Core.csproj]: We can change this uri

**Review by Ander02 — APPROVED** [review]

File: src/Core/Paginator.Core.csproj (hunk 2 above)
Comment (patrickhacens) [reply src/Core/Paginator.Core.csproj]: We should probably cross check the other libraries to see if they have the wrong Url as well

File: src/Core/PaginationData.cs (hunk 1 above)
Comment (patrickhacens) [reply src/Core/PaginationData.cs:24]: Receive generic types and return concrete types
In this case we are making sure this is being converted to an array on our side which means a better performance overall, and for consumers of this API arrays implement IEnumerable<T>

**Pull request merged**